content/en/docs/Writing policies/match-exclude.md

```diff
@@ -134,8 +138,8 @@ spec:
           kinds: # Required, list of kinds
           - Deployment
           - StatefulSet
-          # Optional resource name. Supports wildcards (* and ?)
-          name: "mongo*"
+          # Optional resource names. Supports wildcards (* and ?)
+          names: ["mongo*", "postgres*"]
```

Comment:
same comment as above. YAML list is preferred rather than collection/JSON-style array